Lombiq.Privacy.Samples/Recipes/Lombiq.Privacy.Samples.Content.recipe.json

```diff
@@ -0,0 +1,504 @@
+                  "Expression": "{\r\n  \"DisplayText\": \"{{ Request.Form[\"full-name\"] }}\",\r\n  \"Competitor\": {\r\n    \"Fullname\": {\r\n      \"Text\": \"{{ Request.Form[\"full-name\"] }}\"\r\n    },\r\n    \"Age\": {\r\n      \"Value\": {{ Request.Form[\"age\"] }}\r\n    },\r\n    \"Emailaddress\": {\r\n      \"Text\": \"{{ Request.Form[\"e-mail\"] }}\"\r\n    }\r\n  },\r\n  \"TitlePart\": {\r\n    \"Title\": \"{{ Request.Form[\"full-name\"] }}\"\r\n  }\r\n}"
+                },
+                "ActivityMetadata": {
+                  "Title": "Create competitor item"
+                }
+              }
+            },
+            {
+              "ActivityId": "46bbnrx4k6wbw197vpmdz212dz",
+              "Name": "HttpRedirectTask",
+              "X": 1050,
+              "Y": 160,
+              "IsStart": false,
+              "Properties": {
+                "ActivityMetadata": {
+                  "Title": "Redirect to details"
+                },
+                "Location": {
+                  "Expression": "{{ Workflow.Properties[\"ContentItem\"] | display_url }}"
+                },
+                "Permanent": false
+              }
+            },
+            {
+              "ActivityId": "4n7zznh3ssm9078wyva0rq45d2",
+              "Name": "HttpRedirectTask",
+              "X": 1120,
+              "Y": 30,
+              "IsStart": false,
+              "Properties": {
+                "ActivityMetadata": {
+                  "Title": null
+                },
+                "Location": {
+                  "Expression": "/error"
+                },
+                "Permanent": false
+              }
+            }
+          ],
+          "Transitions": [
+            {
+              "Id": 0,
+              "SourceActivityId": "45g5pkc6tpy86v71mdxq4685d8",
+              "SourceOutcomeName": "Done",
+              "DestinationActivityId": "4bnngn8hxa684201xsaq9fm1wh"
+            },
+            {
+              "Id": 0,
+              "SourceActivityId": "4bnngn8hxa684201xsaq9fm1wh",
+              "SourceOutcomeName": "Valid",
+              "DestinationActivityId": "474fg602kmn9z0xsn389w51mnf"
+            },
+            {
+              "Id": 0,
+              "SourceActivityId": "474fg602kmn9z0xsn389w51mnf",
+              "SourceOutcomeName": "Done",
+              "DestinationActivityId": "46bbnrx4k6wbw197vpmdz212dz"
+            },
+            {
+              "Id": 0,
+              "SourceActivityId": "4bnngn8hxa684201xsaq9fm1wh",
+              "SourceOutcomeName": "Invalid",
+              "DestinationActivityId": "4n7zznh3ssm9078wyva0rq45d2"
+            },
+            {
+              "Id": 0,
+              "SourceActivityId": "474fg602kmn9z0xsn389w51mnf",
+              "SourceOutcomeName": "Failed",
+              "DestinationActivityId": "4n7zznh3ssm9078wyva0rq45d2"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "name": "content",
```

Comment:
```suggestion
      "name": "Content",
```

Reply:
This is lower case in documentation. See [here](https://docs.orchardcore.net/en/dev/docs/reference/modules/Recipes/#content-step)

Reply:
If you look around on the same page ["feature"](https://docs.orchardcore.net/en/dev/docs/reference/modules/Recipes/#feature-step) is lower case too, yet you used "Feature" in the recipe. I'm pretty sure it's case-insensitive. (I have just verified that using "Content" works.) We should make it consistent and all the other step names are PascalCase so this should be as well.